cmd/helpers/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
We got rid of bazel in this repo so this PR will need to be updated to remove bazel as well.

Reply:
@nikhita, I've removed the bazel references & also rebased the PR again. PTAL.